requirements.txt

```diff
@@ -3,22 +3,23 @@ decorator==4.0.10
 docutils==0.12
 ebaysdk==2.1.5
 feedparser==5.2.1
-gevent==1.1.2
+gevent==1.1.2 ; sys_platform != 'win32'
 greenlet==0.4.10
 html2text==2016.9.19
 Jinja2==2.8
-lxml==3.5.0
+lxml==3.7.1 ; sys_platform != 'win32'
+lxml ; sys_platform == 'win32'
 Mako==1.0.4
 MarkupSafe==0.23
 mock==2.0.0
 num2words==0.5.4
 ofxparse==0.16
 passlib==1.6.5
-Pillow==3.4.1
-psutil==4.3.1
+Pillow==4.0.0
+psutil==4.3.1; sys_platform != 'win32'
 psycopg2==2.7.3.1
```

Comment:
`psycopg2==2.7.3.1; sys_platform != 'win32'` while we're at it, then remove from the doc the part that says to edit the requirements.txt file and ta-dah!

Reply:
I think that psutil can be bumped to 5.0.1 to stick to the Debian Stretch version:
https://packages.debian.org/stretch/python3-psutil